The first part is for GTK and makes sure that the port finds the right OpenGL library
The second part makes sure that install_name_tool doesn't break library names and links (http://trac.wxwidgets.org/ticket/15452)
--- configure.orig
+++ configure
@@ -22038,48 +22038,7 @@ ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
 
 
-SEARCH_INCLUDE="\
-    /usr/local/include        \
-    /usr/local/X11/include    \
-    /usr/local/include/X11    \
-    /usr/local/X11R7/include  \
-    /usr/local/X11R6/include  \
-    /usr/local/include/X11R7  \
-    /usr/local/include/X11R6  \
-                              \
-    /usr/Motif-2.1/include    \
-    /usr/Motif-1.2/include    \
-    /usr/include/Motif1.2     \
-                              \
-    /usr/dt/include           \
-    /usr/openwin/include      \
-                              \
-    /usr/include/Xm           \
-                              \
-    /usr/X11R7/include        \
-    /usr/X11R6/include        \
-    /usr/X11R6.4/include      \
-                              \
-    /usr/include/X11R7        \
-    /usr/include/X11R6        \
-                              \
-    /usr/X11/include          \
-    /usr/include/X11          \
-                              \
-    /usr/XFree86/include/X11  \
-    /usr/pkg/include          \
-                              \
-    /usr/local/X1R5/include  \
-    /usr/local/include/X11R5  \
-    /usr/X11R5/include        \
-    /usr/include/X11R5        \
-                              \
-    /usr/local/X11R4/include  \
-    /usr/local/include/X11R4  \
-    /usr/X11R4/include        \
-    /usr/include/X11R4        \
-                              \
-    /usr/openwin/share/include"
+SEARCH_INCLUDE="@@PREFIX@@/include"
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for libraries directories" >&5
 $as_echo_n "checking for libraries directories... " >&6; }
@@ -29402,24 +29361,6 @@ rm -f core conftest.err conftest.$ac_objext \
       ;;
 
       *-*-darwin* )
-        install_name_tool=`which ${HOST_PREFIX}install_name_tool`
-        if test "$install_name_tool" -a -x "$install_name_tool"; then
-            DYLIB_RPATH_POSTLINK="${HOST_PREFIX}install_name_tool -id \$@ \$@"
-            cat <<EOF >change-install-names
-#!/bin/sh
-libnames=\`cd \${2} ; ls -1 | grep '\.[0-9][0-9]*\.dylib\$'\`
-for i in \${libnames} ; do
-    ${HOST_PREFIX}install_name_tool -id \${3}/\${i} \${1}/\${i}
-    for dep in \${libnames} ; do
-        ${HOST_PREFIX}install_name_tool -change \${2}/\${dep} \${3}/\${dep} \${1}/\${i}
-    done
-done
-EOF
-            chmod +x change-install-names
-            DYLIB_RPATH_INSTALL="\$(wx_top_builddir)/change-install-names \${DESTDIR}\${libdir} \$(wx_top_builddir)/lib \${libdir}"
-        fi
-
-                                        HEADER_PAD_OPTION="-headerpad_max_install_names"
       ;;
 
       *-*-cygwin* | *-*-mingw32* )
